src/domain/executable.ts

```diff
@@ -132,7 +132,11 @@ export abstract class ExecutableManager {
     isPersistent = type === EntityType.Instance ? true : isPersistent
 
     const basePrice =
-      paymentMethod === PaymentMethod.Hold ? (isPersistent ? 2_000 : 200) : 0.11
+      paymentMethod === PaymentMethod.Hold
+        ? isPersistent
+          ? 1_000
+          : 100
```

Comment:
```suggestion
      paymentMethod === PaymentMethod.Hold
        ? isPersistent
          ? 1_000
          : 200
```
The pricing should only affect to instances or persistent programs, not to ephemeral programs.